shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -132,22 +132,11 @@ - (void)userNotificationCenter:(UNUserNotificationCenter*)center
   }
 }
 
-static BOOL IsDeepLinkingEnabled(NSDictionary* infoDictionary) {
-  NSNumber* isEnabled = [infoDictionary objectForKey:@"FlutterDeepLinkingEnabled"];
-  if (isEnabled) {
-    return [isEnabled boolValue];
-  } else {
-    return NO;
-  }
-}
-
-- (BOOL)application:(UIApplication*)application
-            openURL:(NSURL*)url
-            options:(NSDictionary<UIApplicationOpenURLOptionsKey, id>*)options
-    infoPlistGetter:(NSDictionary* (^)())infoPlistGetter {
-  if ([_lifeCycleDelegate application:application openURL:url options:options]) {
-    return YES;
-  } else if (!IsDeepLinkingEnabled(infoPlistGetter())) {
+- (BOOL)openURL:(NSURL*)url {
+  NSNumber* isDeepLinkingEnabled =
```

Comment:
I was able to get rid of the static function here.  `-[NSBundle objectForInfoDictionaryKey:]` is now mocked in the tests.